Web/starter_project_vanguard/components/home/HeroSection.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import Link from 'next/link'
+import Image from 'next/image'
+import { BsArrowRightShort } from 'react-icons/bs'
+
+import landing_image from '../../public/img/home-page/landing-image.png'
```

Comment:
[nit] use kabab-case for your imports as well

Reply:
when landing_image is renamed to landing-image it raises error.

Reply:
use camelCase if its an image